ttf2ugui.c

```diff
@@ -156,9 +156,9 @@ static void dumpFont(const UG_FONT * font, const char* fontFile, float fontSize,
 
 
   fprintf(out, "// For copyright, see original font file.\n");
-  fprintf(out, "\n#include \"ugui.h\"\n\n");
+  fprintf(out, "\n#include \"ui/ugui/ugui.h\"\n\n");
```

Comment:
Why don't we just put `ui/ugui` into our includes folders instead?

Why is this change even necessary, it seems to have worked before?
 
Please always write some words into the commit message body to explain basic stuff :pray:

Reply:
This is just a tool for generating the font c sources. We have manually made these changes to the font files previously. Fixing it in the tool makes us have to do less post-processing of the font files.